### ISSUE TITLE
:FZF doesn't work in neovim/iTerm2

### ISSUE THREAD

**Issue by chalmagean** [junegunn/fzf.vim#58]
Hey guys, I get the following in neovim (iTerm2 - Build 2.9.20151117-nightly) when running `:FZF`

```
:FZF
 [H [2J
 [?1049h [1;24r (B [m [4l [?7h [?1000h [39;49m [39;49m (B [m [H [2J [24d (B [0;1m [38;5;110m>  [39m (B [m [A [38;5;144m0/0Assertion failed: (new_col >= 0), function ui_cursor_left, file /tmp/neovim20151222-20363-1q3zv6n/neovim-0.1.1/src/nvim/ui.c, line 450.
Abort trap: 6
```

```
$ nvim -v
NVIM 0.1.1 (compiled Dec 22 2015 21:35:20)
Build type: RelWithDebInfo
Compilation: /usr/local/Library/ENV/4.3/clang -Wconversion -O2 -g -DDISABLE_LOG -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wvla -fstack-protector-strong -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -DHAVE_CONFIG_H -I/tmp/neovim20151222-20363-1q3zv6n/neovim-0.1.1/build/config -I/tmp/neovim20151222-20363-1q3zv6n/neovim-0.1.1/src -I/tmp/neovim20151222-20363-1q3zv6n/neovim-0.1.1/deps-build/usr/include -I/tmp/neovim20151222-20363-1q3zv6n/neovim-0.1.1/deps-build/usr/include -I/tmp/neovim20151222-20363-1q3zv6n/neovim-0.1.1/deps-build/usr/include/luajit-2.0 -I/tmp/neovim20151222-20363-1q3zv6n/neovim-0.1.1/deps-build/usr/include -I/tmp/neovim20151222-20363-1q3zv6n/neovim-0.1.1/deps-build/usr/include -I/tmp/neovim20151222-20363-1q3zv6n/neovim-0.1.1/deps-build/usr/include -I/tmp/neovim20151222-20363-1q3zv6n/neovim-0.1.1/deps-build/usr/include -I/usr/local/opt/gettext/include -I/usr/include -I/usr/include -I/tmp/neovim20151222-20363-1q3zv6n/neovim-0.1.1/build/src/nvim/auto -I/tmp/neovim20151222-20363-1q3zv6n/neovim-0.1.1/build/include

Optional features included (+) or not (-): +acl   +iconv    +jemalloc
For differences from Vim, see :help vim-differences

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/Cellar/neovim/0.1.1/share/nvim"
```

**Comment by junegunn** [comment]
So neovim crashed with signal 6 (abrt)? That should be a bug of neovim.

**Comment by chalmagean** [comment]
Fixed by upgrading to fzf 0.11.1
